lhotse/recipes/librilight.py

```diff
@@ -0,0 +1,124 @@
+"""
+About the librilight corpus
+Libri-light is a benchmark for the training of automatic speech recognition (ASR) systems with limited or no supervision.
+It contains a large dataset of 60K hours of unlabelled speech from audiobooks in English and a small labelled dataset (10h, 1h, and 10 min) plus metrics, trainable baseline models, and pretrained models that use these datasets.
+It is covered in more detail at https://arxiv.org/abs/1912.07875.
+"""
```

Comment:
If we're not adding download functions, could you at least provide a link to where this dataset can be obtained from?

Reply:
Sure.